frontend/utils/filters.js

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
There is already a filters file in `frontend/views/utils/filters.js`, please move this there

Reply:
Okay.